tock/hours/models.py

```diff
@@ -325,10 +325,21 @@ def calculate_target_hours(self):
     def calculate_total_allocation_hours(self):
         """
         Calculates an estimate of the hours spent on projects with a weekly billable
-        allocation by multiplying that allocation against this timecard's target hours.
-        """
+        allocation.
+
+        This calculation does not use self.target_hours, which incorporates
+        self.billable_expectation, because the allocation % indicated by users
+        is a reflection of the proportion of their total week, not as a proportion
+        of their expected billable hours.
```

Comment:
Yes, this explanation of WB allocation instead of hourly expectation aligns with my understanding too.